README.md

```diff
@@ -215,6 +215,14 @@ The possible values for the annotation can be
 * `"my-instrumentation"` - name of `Instrumentation` CR instance.
 * `"false"` - do not inject
 
+It as also possible in case of multiple-containers Pods (like Istio sidecars) to specify the name of the container in which you want to inject OTEL auto-instrumentation (by default the first container is choosen)
+
+For this add this annotation to your Pod Specification with target container name :
+
+```bash
+instrumentation.opentelemetry.io/container-name: "name_of_your_target_container"
```

Comment:
@fscellos could you please rename it to `instrumentation.opentelemetry.io/container-names` and support list as well?

Reply:
Hello @pavolloffay. I will try to do it at worst next week, yes.